package.json

```diff
@@ -72,7 +72,7 @@
     "build": "npm run clean && tsc",
     "build-demo": "npm run clean && webpack --config webpack.config.demo.js --mode production",
     "clean": "rimraf dist",
-    "deploy": "gh-pages -d example/dist",
```

Comment:
Run the `npm run clean` script here also because just like the publish script, we need a clean dist folder to be generated ❗

Reply:
Hmm, I'm not so sure about that, as we don't use this script separately, but only as part of the `publish-demo` script where we delete the `dist` folder at the `build` stage. Perhaps, we can remove the `deploy` script and be more specific in the `publish-demo`:

```
npm run build-demo && gh-pages -d dist
``` 

@jamesrweb What do you think about that?

Reply:
You're right, the scripts need updated and cleaned but for now this is working as expected. Good work!